blog_test.ts

```diff
@@ -5,8 +5,8 @@ import {
   assert,
   assertEquals,
   assertStringIncludes,
-} from "https://deno.land/std@0.153.0/testing/asserts.ts";
```

Comment:
version **0.154.0** can not work in deploy currently, please keep 0.153 or wait 0.155, check https://github.com/denoland/deno_std/pull/2614